R/surv-roc_survival_curve.R

```diff
@@ -140,4 +164,27 @@ roc_curve_survival_impl <- function(truth,
 }
 
 # Dynamically exported
-autoplot.roc_survival_df <- autoplot.roc_df
+autoplot.roc_survival_df <- function(object, ...) {
+
+  `%+%` <- ggplot2::`%+%`
+
+  # Base chart
+  roc_chart <- ggplot2::ggplot(data = object)
+
+  # create aesthetic
+  roc_aes <- ggplot2::aes(
+    x = 1 - specificity,
+    y = sensitivity,
+    color = .eval_time
+  )
+
+  # build the graph
+  roc_chart <- roc_chart %+%
+    ggplot2::geom_path(mapping = roc_aes) %+%
```

Comment:
Use `geom_step(direction = "hv")` instead of a line.

Reply:
done in [`0826829` (#407)](https://github.com/tidymodels/yardstick/pull/407/commits/0826829a016e37a5d0f7c55073b52da45868cc07)